### PR TITLE
Local cloud config

### PR DESCRIPTION
**Pull request by rimusz**
How easily to host cloud-config locally

### CODE REVIEW

**Comment by chancez** [comment]
@rimusz Do you want to squash those commits together perhaps?

File: README.md [hunk 1]
```diff
@@ -110,14 +110,16 @@ through the following environment variables:
...
-  > `CLOUD_CONFIG=http://192.168.64.1/~am/coreos-xhyve/xhyve.cloud-init ./coreos-xhyve-run`
+  How to host cloud-config locally:
+  ```
+  cd cloud-init 
+  python -m SimpleHTTPServer >/dev/null 2>&1 &
+  cd ..
+  ```
+  Usage would be something like:
+  ```
+  CLOUD_CONFIG=http://192.168.64.1:8000/user-data ./coreos-xhyve-run
```
Comment (chancez) [inline README.md:121]: Would need to be `sudo CLOUD_CONFIG=http://192.168.64.1:8000/user-data ./coreos-xhyve-run`

Comment (rimusz) [inline README.md:121]: yes
sorry forgot to include sudo

sudo CLOUD_CONFIG=http://192.168.64.1:8000/user-data ./coreos-xhyve-run

Comment (chancez) [inline README.md:121]: No, the env variable needs to happen after sudo, because sudo does not pass through environmental variables.

**Pull request closed without merging**